frontend/test/metabase/scenarios/smoketest/admin_setup.cy.spec.js

```diff
@@ -370,10 +370,13 @@ describe("smoketest > admin_setup", () => {
     });
 
     it("should modify user permissions for data access and SQL queries, both on a database/schema level as well as at a table level as admin", () => {
-      // *** Need multible databases to test their permissions.
-
       cy.signOut();
       cy.signInAsAdmin();
+
+      cy.request("POST", "/api/permissions/group", {
+        name: "Marketing",
+      });
+
```

Comment:
This kind of interdependency and sequential flow is exactly the reason why these tests are bad and why they need to be removed.

This test was relying on the "Marketing" group created In the previous (now deleted) `describe` block.